### ISSUE TITLE
.net Core

### ISSUE THREAD

**Issue by eworksegypt** [netTiers/netTiers#488]
how to upgrade to .net core?

**Comment by bildermann** [comment]
I have been playing around with a version I stripped down to experiment with.  Been using Codesmith and netTiers for about 15 years but never really looked into internals because nevere had any significant issue with it love it.

First issue you will probably find is that the Identity code will not work as is, so that would need to be fixed, the next would be (ideally!) to strip out reliance on anything to do with current EF used, I hate the fact it's included when there's such a large dependency on something for so little - Wish there was a way to generate the code without this dependency even in current version - maybe there is but I haven't been able to find a way.

**Comment by jeremeguenther** [comment]
I agree, I would love to remove all remaining references to EntLib.  So far I have removed the Caching dependency and the Error handling dependency.  They can now be turned off via the "02. Framework Generation - Optional" settings.  Since I don't use netTiers in any .net Core development I haven't had a reason to strip out the rest of the references, I am not sure if or what they are still being used for.
